tests/test_map.py

```diff
@@ -1436,6 +1436,11 @@ def test_map_pickle(self):
     def test_map_is_subscriptable(self):
         self.assertIs(self.Map[int, str], self.Map)
 
+    def test_kwarg_named_col(self):
+        self.assertEqual(dict(self.Map(col=0)), {"col": 0})
+        self.assertEqual(dict(self.Map(a=0, col=1)), {"a": 0, "col": 1})
+        self.assertEqual(dict(self.Map({"a": 0}, col=1)), {"a": 0, "col": 1})
```

Comment:
All of these three asserts fail when running in `PyMapTest`. They now pass with the changes in this MR.

This test makes very little sense now that the keyword arg called "col" has been refactored out, though. Should we keep it or not?

Reply:
It's OK to keep it I guess.